### DISCUSSION TITLE
How to chance icon at the side of the link in link bar?

### DISCUSSION THREAD

**Discussion by moraesfernanda** [greenelab/lab-website-template#89] [Q&A] [answered]
Chance the icon at the side of the link in the link bar

Is it possible to change the icon at the link bar? We already changed the logo, but it was not enough.

https://metabio.netlify.app/
Thank you very much!

**Comment by vincerubinetti** [comment, marked as answer]
Are you talking about this:

![image](https://user-images.githubusercontent.com/8326331/127377958-3d4f6181-d424-449b-9095-e072d9fbf483.png)

If so, this page explains it:

https://github.com/greenelab/lab-website-template/wiki/Basic-Editing#favicons

You basically just replace the images in the `/favicons` folder with your own.

**Reply by moraesfernanda** [reply]
Thank you!!! I missed this reply! I will try it.
